### ISSUE TITLE
New binding - Azure SQL Server

### ISSUE THREAD

**Issue by edevil** [Azure/azure-webjobs-sdk-extensions#85]
New feature - An Azure SQL server binding with a supplied query.

**Comment by mathewc** [comment]
Yes, we've discussed adding both a SQL trigger as well as input/output bindings. It's been on the radar for a while :)

**Comment by MV10** [comment]
The ability to trigger a Function when a row is added is apparently already available in preview. Unfortunately it doesn't appear to be available if you create the Function from Visual Studio. (Does anybody actually do _real_ work in the feature-free Functions editor in the portal???)

Can anyone comment on when this will be out of preview, and when we can expect it to be added to the VStudio side of the fence? 

https://stackoverflow.com/a/43881107/152997

https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-external-table
